Rego/Utils/ReportUtils.rego

```diff
@@ -15,45 +15,40 @@ ScubaBaseUrl := sprintf("https://github.com/cisagov/ScubaGear/blob/%v/baselines/
 
 PolicyAnchor(PolicyId) := sprintf("#%v", [replace(lower(PolicyId), ".", "")])
 
-PolicyProduct(PolicyId) := product if {
-    dotIndexes := indexof_n(PolicyId, ".")
-    product := lower(substring(PolicyId, 3, dotIndexes[1]-dotIndexes[0]-1))
+PolicyProduct(PolicyId) := Product if {
+    DotIndexes := indexof_n(PolicyId, ".")
+    Product := lower(substring(PolicyId, 3, (DotIndexes[1] - DotIndexes[0]) - 1))
```

Comment:
I am not sure what changes were made by you explicitly or perhaps by the opa formatter.  Here the change appears to be a style change to have a space around operators.  Add to style guide?

Reply:
I changed the first letter in the variable names to be capitalized because we are using pascal case